cookbooks/aws-parallelcluster-common/resources/network_service/network_service_ubuntu1804.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License
+provides :network_service, platform: 'ubuntu', platform_version: '18.04'
+unified_mode true
+default_action :restart
+
+use 'partial/_restart_network_debian'
+
+action :restart do
+  action_restart_network
+end
+
+action :reload do
+  action_apply_network_changes
+end
```

Comment:
Nitpick. Looks like common code which could go directly into `_restart_network_debian`.

Reply:
you're right. I can move it.

Reply:
Fixed.